src/applications/mhv-landing-page/tests/e2e/mhv-landing-page.cypress.spec.js

```diff
@@ -1,9 +1,38 @@
-import manifest from '../../manifest.json';
+import { appName, rootUrl } from '../../manifest.json';
+import user from '../fixtures/user.json';
+import vamcEhr from '../fixtures/vamc-ehr.json';
+import { generateFeatureToggles } from '../../mocks/api/feature-toggles';
+import ApiInitializer from './utilities/ApiInitializer';
 
-describe(manifest.appName, () => {
-  it('Visit bad URL', () => {
-    cy.visit(`${manifest.rootUrl}/dummy`);
+/*
+ * The intent of the tests in this file is to replicate the current state
+ * of the landing page as it is in production.
+ * As of 2/14/2024, the landing page is enabled, but the personalization is not.
+ */
+describe(`${appName} - landing page`, () => {
+  beforeEach(() => {
+    cy.intercept(
+      'GET',
+      '/v0/feature_toggles*',
+      generateFeatureToggles({
+        mhvLandingPageEnabled: true,
+        mhvLandingPagePersonalization: false,
+      }),
+    ).as('featureToggles');
+    cy.intercept('GET', '/data/cms/vamc-ehr.json', vamcEhr).as('vamcEhr');
+  });
+
+  it('display the landing page and welcome message', () => {
+    ApiInitializer.initializeMessageData.withNoUnreadMessages();
+    cy.intercept('GET', '/v0/feature_toggles*', generateFeatureToggles()).as(
+      'featureToggles',
+    );
+    cy.login(user);
+    cy.visit(rootUrl);
     cy.injectAxeThenAxeCheck();
-    cy.findByRole('heading', { name: /we can’t find that page/ }).should.exist;
+    cy.findByRole('heading', { name: /^My HealtheVet$/i }).should.exist;
+    cy.findByRole('heading', { level: 2, name: /^Welcome/ }).should(
```

Comment:
The test is named `'display the landing page and welcome message'`but the test looks for the absence of the welcome message. The name or assertion needs updating, I believe.

Also, is this redundant with the tests you added in `src/applications/mhv-landing-page/tests/e2e/personalization/welcome-message.cypress.spec.js`?

Reply:
I changed the name